tests/test_example_2.py

```diff
@@ -136,10 +136,10 @@ def test_train_two():
         batch_size=len(df_train),
         num_passes=10,
         weight_optimization=AnalyticOptimizationParameters(),
-        get_states_extra=LOBPCGFastStateParameters(iterations=10),
+        get_states_extra=LOBPCGFastStateParameters(iterations=20),
     )
 
-    qcml.wait_for_training()
+    qcml.wait_for_training(poll_time=10)
```

Comment:
Why the `poll_time`?

Reply:
It's a name parameter that has a default value of 60 seconds. During the examples, reducing the poll time to 10 seconds on short trainings, is more effective in retrieving metrics about the training progress